frontend/src/components/FilterButton.tsx

```diff
@@ -0,0 +1,12 @@
+export default function FilterButton({ filterLabel }: { filterLabel: string }) {
```

Comment:
This is me being nitpicky. I feel like the prop name `filterLabel` is redundant as the Component is already called FilterButton.  I would prefer just using `label`